charts/openshift-metering/templates/hive/hive-metastore-statefulset.yaml

```diff
@@ -283,6 +287,11 @@ spec:
           name: hive-jmx-config
       # these emptyDir volumes are necessary because Openshift requires VOLUMEs
       # in a Dockerfile have a corresponding volumeMount
+{{ if .Values.hive.spec.config.db.secretName }}
+      - name: hive-metastore-db-secret
+        secret: metastore-db-secret
+          secretName: metastore-secret
```

Comment:
```suggestion
          secretName: "{{ .Values.hive.spec.config.db.secretName }}"
```

Reply:
Yep, can't lock the users down to a specific name. Fixed